fiorella/src/styles/layout/_header.scss

```diff
@@ -0,0 +1,247 @@
+/*================ Site Header ================*/
+.homepage-logo {
+  display: none;
+}
+
+.desktop-btn-search {
+  display: none;
+}
+
+.header-buttons-container {
+  display: none;
+}
+
+.menu-container {
+  background-color: $color-white-solid;
+  font-family: $font-menu-header;
+  height: 100%;
+  width: 0;
+  overflow-x: hidden;
+  position: fixed;
+  transition: 0.5s;
+  z-index: 2;
+
+  a {
+    color: $color-black-solid;
+    text-decoration: none;
+    font-size: 25px;
+  }
+
+  @include reset-ul(block);
+}
+
+.show-menu {
+  width: 100%;
+  padding: 0 20px ;
+}
+
+.menu-top-section {
+  display: flex;
+  justify-content: center;
+  padding-top: 10px;
+  position: relative;
+
+  .menu-close {
+    @extend %positioning;
+    justify-content: flex-end;
+  }
+
+  button {
+    background-color: transparent;
+    border-style: none;
+  }
+}
+
+.search-container {
+  border-bottom-style: solid;
+  border-bottom-color: $color-featured;
+  margin: 20px 0 5px 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `20px 0 5px` instead of `20px 0 5px 0`